node/src/cli.rs

```diff
@@ -61,7 +61,7 @@ pub enum ProviderType {
 #[derive(ValueEnum, Clone, Debug)]
 pub enum StorageLayer {
     /// RocksDB
-    Rocksdb,
+    RocksDB,
```

Comment:
Reason why I initially named this with lowercase "db" is because the `ValueEnum` macro separates the string version of the command based on camel case. Not sure if it will work but before I remember naming it `RocksDb` and it would run with `rocks-db`.

Reply:
You're right, from running `--help` we see that:
```
Type of StorageHub provider [default: memory] [possible values: rocks-db, memory]
```
Anyway, I don't think this is bad.